homeassistant/components/mikrotik/hub.py

```diff
@@ -101,10 +101,18 @@ def get_hub_details(self) -> None:
         self.model = self.get_info(ATTR_MODEL)
         self.firmware = self.get_info(ATTR_FIRMWARE)
         self.serial_number = self.get_info(ATTR_SERIAL_NUMBER)
-        self.support_capsman = bool(self.command(MIKROTIK_SERVICES[IS_CAPSMAN]))
-        self.support_wireless = bool(self.command(MIKROTIK_SERVICES[IS_WIRELESS]))
-        self.support_wifiwave2 = bool(self.command(MIKROTIK_SERVICES[IS_WIFIWAVE2]))
-        self.support_wifi = bool(self.command(MIKROTIK_SERVICES[IS_WIFI]))
+        self.support_capsman = bool(
+            self.command(cmd=MIKROTIK_SERVICES[IS_CAPSMAN], discover=True)
+        )
+        self.support_wireless = bool(
+            self.command(cmd=MIKROTIK_SERVICES[IS_WIRELESS], discover=True)
+        )
+        self.support_wifiwave2 = bool(
+            self.command(cmd=MIKROTIK_SERVICES[IS_WIFIWAVE2], discover=True)
+        )
+        self.support_wifi = bool(
+            self.command(cmd=MIKROTIK_SERVICES[IS_WIFI], discover=True)
```

Comment:
```suggestion
        self.support_capsman = bool(
            self.command(MIKROTIK_SERVICES[IS_CAPSMAN], discover=True)
        )
        self.support_wireless = bool(
            self.command(MIKROTIK_SERVICES[IS_WIRELESS], discover=True)
        )
        self.support_wifiwave2 = bool(
            self.command(MIKROTIK_SERVICES[IS_WIFIWAVE2], discover=True)
        )
        self.support_wifi = bool(
            self.command(MIKROTIK_SERVICES[IS_WIFI], discover=True)
```

`cmd` is a positional parameter, so no need to add the `cmd=`